packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -65,6 +65,20 @@
 	margin-top: -1px; /* To collapse the margin with the previous element */
 }
 
+.font-library-modal__font-variant_demo-image {
+	display: block;
+	height: $grid-unit-30;
+	width: auto;
+}
+
+.font-library-modal__font-variant_demo-text {
+	white-space: nowrap;
+	flex-shrink: 0;
+	font-size: 18px;
+	transition: opacity 0.3s ease-in-out;
+	@include reduce-motion("transition");
```

Comment:
We probably need to include the transition into the demo-image class, too, to apply the same transition when the font has and lacks an image preview.

Reply:
Yep consistency would be nice. Unrelated to the scope of this PR though.